src/main/java/apoc/refactor/GraphRefactoring.java

```diff
@@ -396,4 +407,23 @@ private Relationship copyRelationship(Relationship rel, Node source, Node target
         Relationship newrel = startNode.createRelationshipTo(endNode, rel.getType());
         return copyProperties(rel, newrel);
     }
+
+
+
+    public void mergeRelsWithSameTypeAndDirectionInMergeNodes (Node node, RefactorConfig config, Direction dir){
+        List<Relationship> rels = new ArrayList<>();
```

Comment:
this code is really weird, please fix it
use foreach
iterate over the iterator